python_scripts/trees_classification.py

```diff
@@ -169,14 +172,16 @@ def plot_decision_function(fitted_classifier, range_features, ax=None):
 # class predicted when the culmen depth is inferior to the threshold.
 
 # %%
-tree.predict([[0, 15]])
+sample_1 = pd.DataFrame([[0, 15]], columns=culmen_columns)
```

Comment:
It could be clearer to make a dict:

```suggestion
sample_1 = pd.DataFrame({
    "Culmen Length (mm)": 0,
    "Culmen Depth (mm)": 15,
})
```